gui/scripts/syncthing/core/controllers/syncthingController.js

```diff
@@ -1157,7 +1157,11 @@ angular.module('syncthing.core')
         };
 
         $scope.rescanFolder = function (folder) {
-            $http.post(urlbase + "/scan?folder=" + encodeURIComponent(folder));
```

Comment:
I'd rather see this as a separate function than with an optional parameter, tbh...